core/pom.xml

```diff
@@ -418,17 +418,7 @@ THE SOFTWARE.
     </dependency>
     <dependency>
       <groupId>org.mockito</groupId>
-      <artifactId>mockito-core</artifactId>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.powermock</groupId>
-      <artifactId>powermock-module-junit4</artifactId>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.powermock</groupId>
-      <artifactId>powermock-api-mockito2</artifactId>
+      <artifactId>mockito-inline</artifactId>
       <scope>test</scope>
     </dependency>
```

Comment:
The corresponding dependencies should be removed from the `<dependencyManagement>` section of the root `pom.xml`, along with the `powermock.version` property in the `<properties>` section.

Reply:
Can you link to somewhere I missed?

I can't find anything in a search:

```
➜  jenkins git:(nuke-powermock) rg -i powermock
➜  jenkins git:(nuke-powermock) rg -i mockito-core
```

or in https://github.com/jenkinsci/pom/blob/master/pom.xml